README.md

```diff
@@ -1,41 +1,54 @@
-# nix-config
+# nix-config ![CI](https://github.com/lovesegfault/nix-config/workflows/CI/badge.svg)
 
-This repository holds both my NixOS configuration (`system/`) as well as my Home
-Manager configuration (`home/`), both work together to generate fully
-configured, consistent, systems to my liking.
+This repository holds both my NixOS configuration. It is fully reproducible
```

Comment:
```suggestion
This repository holds my NixOS configuration. It is fully reproducible
```

(reflow as needed)

Alternatively (hopefully not putting words in your mouth):
```suggestion
This repository holds both my NixOS and Home Manager configuration. It is fully reproducible
```